source/common/init/manager_impl.cc

```diff
@@ -53,16 +55,30 @@ void ManagerImpl::initialize(const Watcher& watcher) {
     // completed immediately.
     for (const auto& target_handle : target_handles_) {
       if (!target_handle->initialize(watcher_)) {
-        onTargetReady();
+        onTargetReady(target_handle->name());
       }
     }
   }
 }
 
-void ManagerImpl::onTargetReady() {
+// For init manager to query unready targets.
+void ManagerImpl::checkUnreadyTargets() {
+  ENVOY_LOG(debug, "QUERY: List of unready targets..................");
```

Comment:
What does "query" mean in this context? Can this log entry just say `List of unready targets:`?